### PR TITLE
fix(text-field): removed --float-above from --shake selectors

### PR DESCRIPTION
**Pull request by moog16**

### CODE REVIEW

File: packages/mdc-textfield/mdc-text-field.scss [hunk 1]
```diff
@@ -186,11 +186,10 @@
   }
 
   .mdc-text-field__label--float-above {
-    animation: invalid-shake-float-above-box 250ms 1;
```
Comment (moog16) [inline packages/mdc-textfield/mdc-text-field.scss]: This bug I introduced last PR, which needs to be removed

File: packages/mdc-textfield/mdc-text-field.scss [hunk 2]
```diff
@@ -187,7 +187,7 @@
     transform: translateY(-50%) scale(.75);
```
Comment (bonniezhou) [inline packages/mdc-textfield/mdc-text-field.scss]: This seems to be a duplicate of the selector just above it.

**Comment by moog16** [comment]
yup totally right...idk how i missed those all. Must have been a merge conflict. :|

File: packages/mdc-textfield/mdc-text-field.scss (hunk 2 above)
Comment (moog16) [reply packages/mdc-textfield/mdc-text-field.scss]: :(

**Pull request merged**